src/cport/modules/cons_ppisp.py

```diff
@@ -84,15 +106,43 @@ def retrieve_prediction_link(self, url=None, page_text=None):
 
     @staticmethod
     def download_result(download_link):
-        """Download the results."""
+        """
+        Download the results.
+
+        Parameters
+        ----------
+        download_link : str
+            The link to the results.
+
+        Returns
+        -------
+        str
```

Comment:
Returns should have the same description as the same function in ispred4.
"temp_file.name : str
             The path to the results file."